app/Photo.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Photo extends Model
+{
+    const DATE_FORMAT_IMAGE_NAME = "Y-m-d-G-i-s";
+    const DATE_FORMAT_SORTING_BY_DATE = "Y/m";
+
+    public static function getCountRows()
+    {
+        return self::get()->count();
+    }
+
+    public static function getAllRows()
+    {
+        return self::get();
+    }
+
+    public static function getLastRecord()
+    {
+        return self::orderBy('id', 'desc')->first();
+    }
+
+    private static function saveImageFromRaspi(\Illuminate\Http\Request $request)
```

Comment:
You do not need to pass whole Request object. Pass only content of $request->file->('photo')

Reply:
fixed